include/deal.II/lac/petsc_ts.templates.h

```diff
@@ -79,27 +85,32 @@ namespace PETScWrappers
 
 
   template <typename VectorType, typename PMatrixType, typename AMatrixType>
-  TS
-  TimeStepper<VectorType, PMatrixType, AMatrixType>::petsc_ts()
+  DEAL_II_CXX20_REQUIRES((concepts::is_dealii_petsc_vector_type<VectorType> ||
+                          std::constructible_from<VectorType, Vec>))
+  TS TimeStepper<VectorType, PMatrixType, AMatrixType>::petsc_ts()
   {
     return ts;
   }
 
 
 
   template <typename VectorType, typename PMatrixType, typename AMatrixType>
+  DEAL_II_CXX20_REQUIRES((concepts::is_dealii_petsc_vector_type<VectorType> ||
+                          std::constructible_from<VectorType, Vec>))
   inline MPI_Comm
-  TimeStepper<VectorType, PMatrixType, AMatrixType>::get_mpi_communicator()
-    const
+    TimeStepper<VectorType, PMatrixType, AMatrixType>::get_mpi_communicator()
```

Comment:
bug in the formatter?

Reply:
Possibly -- who knows. clang-format is a thing unto its own...